gradle-plugin/plugin/build.gradle.kts

```diff
@@ -38,15 +37,9 @@ tasks.withType<KotlinCompile>().configureEach {
 }
 
 // This directory will contain one file per version of the Android Gradle Plugin that we wish to test against.
-val agpClasspathDir =
-  project.layout.buildDirectory.dir("agp-classpath").get().asFile.apply {
-    mkdir()
-  }
+val agpClasspathDir = project.layout.buildDirectory.dir("agp-classpath")
```

Comment:
Now that this is a task output, gradle will automatically create the directory for us simplifying this piece of code.